opentelemetry-semantic-conventions/src/opentelemetry/semconv/_incubating/attributes/cpython_attributes.py

```diff
@@ -0,0 +1,30 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enum import Enum
+from typing import Final
+
+CPYTHON_GC_GENERATION: Final = "cpython.gc.generation"
+"""
+Value of the garbage collector collection generation.
+"""
+
+
+class CpythonGcGenerationValues(Enum):
```

Comment:
we can make it right.  I think PascalCase is doing us a bad favor here. 

I think we can fix it with text mapping like in https://github.com/open-telemetry/opentelemetry-python/blob/adccbc81a4714cd34d39c78a79f477f00d6f7b66/scripts/semconv/templates/registry/weaver.yaml#L41 or a custom-casing like in https://github.com/open-telemetry/weaver/issues/415#issuecomment-2616965937, lmk if you need any help

Reply:
Done, thanks for the hint!